tests/plotters/conftest.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import pytest
+
+from compas.datastructures import Mesh
+
+
+# ==============================================================================
+# Fixtures
+# ==============================================================================
+
+@pytest.fixture
+def start():
+    """
+    3D point as start point.
+    """
+    return [0.0, 1.0, 0.0]
+
+@pytest.fixture
+def direction():
+    """
+    3D vector as direction.
+    """
+    return [1.0, 0.0, 0.0]
+
+@pytest.fixture
+def length():
+    """
+    Length.
+    """
+    return 1.0
+
+@pytest.fixture
+def vector_tag():
+    return "a_vector"
+
+@pytest.fixture
+def meshNoAttr():
```

Comment:
A comment on formatting here. The convention in Python is to use "snake case" (everything lower-cased with underscores to split meaningful words) instead of "camel case" (more popular in Javascript, C# -- uppercased characters at the beginning of every new word. I think we should stick to snake case for consistency. This means, `meshNoAttr` should become something like `mesh_no_attrs` :) This applies to other functions in this file `conftest.py` and in `test__lines.py` (e.g. `test_line_sdl_OneSide` -> `test_line_sdl_one_side`).